src/tasks/lint-locales.js

```diff
@@ -12,9 +12,16 @@ const Reporter = require('./includes/lint-reporter.js').default;
  * @private
  */
 function lintLocales() {
-  return runAll(config.src.root, new Reporter()).then((reporter) => {
-    return reporter.output();
-  });
+  return runAll(config.src.root, new Reporter())
+    .then((reporter) => {
+      return reporter.output();
+    })
+    .catch((err) => {
+      console.log(err);
```

Comment:
Yeah I felt weird about that too / was going to ask you your opinion - thanks for pointing that out.